containers/unit-tests/exec

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh -e
+
+checkargs() { test $# -eq $1; }
+
+filter='label=org.cockpit-project.container=unit-tests'
+defaultargs='-it'
+defaultcmd='/bin/bash'
+args=''
+id=''
+
+while test $# -gt 0; do
+  case "$1" in
+    --root) args="$args -uroot";;
+    --id=*) id="${1#--id=}";;
+    -it) args="$args -it"; defaultargs='';;
```

Comment:
What does that option actually do? `-it` is already the default?

Reply:
this moves `-it` from `defaultargs` to `args`.  `defaultargs` is only used when invoking a shell (see below).